core/templates/domain/utilities/language-util.service.spec.ts

```diff
@@ -83,31 +83,31 @@ describe('Language util service', function() {
 
```

Comment:
(Should we add a test to ensure that the language list has a direction with either ltr or rtl values?)

Reply:
I think so adding a test for the language constants is a good idea.
Where would this go? In `language-util.service.spec.ts`?

Reply:
I added a test verifying this. I wasn't sure if I approached it the right way, see line 186 in `language-util.service.spec.ts`